server/src/main/java/org/candlepin/async/tasks/EntitlerJob.java

```diff
@@ -118,20 +118,19 @@ public Object execute(final JobExecutionContext context) throws JobExecutionExce
         return poolErrors;
     }
 
-    public static EntitlerJobConfig createConfig() {
-        return new EntitlerJobConfig();
+    public static EntitlerJobConfig createConfig(int limit) {
+        return new EntitlerJobConfig(limit);
     }
 
     /**
      * Job configuration object for the entitle by pool job
      */
     public static class EntitlerJobConfig extends JobConfig {
 
-        public EntitlerJobConfig() {
+        public EntitlerJobConfig(int limit) {
```

Comment:
This is fine for now since we don't have a convenient means of updating the limit such that we can safely add it as another chained method; but this is something we should probably change in future work.